### PR TITLE
Add mongodb

### PR DESCRIPTION
**Pull request by frvnkly**
- requires local installation of mongodb
- added mongoose
- express connects to local mongodb service using mongoose
- readme updated

### CODE REVIEW

**Comment by frvnkly** [comment]
Thanks! Did you get it to work? I think that some of the commands might not work on Windows if you're working on one.

File: server/mongoose.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const MONGODB_URI = 'mongodb://localhost:27017/pesto';
```
Comment (skassam21) [inline server/mongoose.js]: This should be a variable that is stored in a config file, and in that config file, it should use environment variables

File: server/app.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ const pingRouter = require("./routes/ping");
 
 const { json, urlencoded } = express;
 
+// connect to MongoDB
+require('./mongoose');
```
Comment (skassam21) [inline server/app.js]: It's confusing that this is called mongoose - same name as the package

Comment (skassam21) [reply server/app.js]: Also all requires should be at the top above the `const { .. `

**Pull request merged**